resources/deu.ini

```diff
@@ -1,208 +1,219 @@
 ; German language pack for MyPlot
 ; Created by SalmonDE and morpheus132
-; Updated time : 12/20/2016
+; Updated time : 07/22/2017
 
 ; Popup when entering a plot
-popup=Du hast den Plot {%0} betreten
-popup.owner=Dieser Plot gehört {%0}
-popup.available=Dieser Plot ist verfügbar! Benutze /p claim um den Plot zu beanspruchen!
-popup.denied=Du darfst diesen Plot nicht betreten!
+popup=Du hast das Grundstück {%0} betreten.
+popup.owner=Dieses Grundstück gehört {%0}.
+popup.available=Dieses Grundstück ist verfügbar! Benutze /p claim, um das Grundstück zu beanspruchen.
```

Comment:
> "Benutze /p claim um das Grundstück zu beanspruchen"

Wouldn't "es" fit better here because the word "Grundstück" is already used in the previous sentence?